### ISSUE TITLE
Does mass=0 freezes the point or just constraint's length?

### ISSUE THREAD

**Issue by vorg** [mattdesl/verlet-constraint#1]
I have 

``` javascript
var point = makePoint([50, 50])
var center = {
    position: [ 0, 0 ],
    mass: 0
}

var c = Constraint([point, center], { stiffness: 0.005, restingDistance: 10 })
c.solve()
```

and now: 

> If one of the points has a mass of zero, that body will be considered "unmovable".

I would assume `c` would never move? But it does.

Looking at the code it seems like you move both points: [build.js#L51](https://github.com/mattdesl/verlet-constraint/blob/master/lib/build.js#L51)

``` javascript
//push/pull based on mass
vec.scale(scaled, delta, scalarP1 * restingRatio)
vec.add(p1vec, p1vec, scaled)

vec.scale(scaled, delta, scalarP2 * restingRatio)
vec.sub(p2vec, p2vec, scaled)
```

**Comment by mattdesl** [comment]
Yes I think the mass calculation is borked (see https://github.com/mattdesl/verlet-system/issues/1) and I'm open to any changes/suggestions to fix it.

**Comment by vorg** [comment]
It should be just simple Newton's second law, right? `F = m * a` -> `a = F / m`. Will have a look. 

What I have found tricky when using `verlet-system` is that it's not clear how to incorporate external forces (e.g. collisions) so I ended up modifying positions outside of the system, same as `verlet-constraint`. If we now fix the mass code in the system, we will need to take mass into account also in constraints and everywhere else. Seems like duplication but maybe in a good way (decoupling).
